src/kaleidoscope/device/dygma/keyboardManager/universalModules/Hand.cpp

```diff
@@ -1,7 +1,7 @@
 #ifdef ARDUINO_ARCH_NRF52
 #include "Hand.h"
 
-#include "Raise2.h"
+#include "kaleidoscope/device/dygma/keyboardManager/KeyboardManager.h"
```

Comment:
Why is this so long? Could we add the path to the Makefile INCDIR and use just #include "KeyboardManager.h"?

Long paths make it later difficult to re-organize the structure if needed. If it is in the Makefile, then we change the path only on one place.